README.md

```diff
@@ -275,6 +275,14 @@ By default, the verifier loads Inspec plugins such as additional Reporter or Inp
       load_plugins: false
 ```
 
+Some Inspec plugins allow further configuration. You can supply these settings as well with InSpec 4.26 or newer:
+
+```yaml
+    verifier:
+      plugin_config:
+        example_setting: "Example value"
```

Comment:
Should this looks like:
```yaml
    verifier:
      plugin_config:
        example_plugin_name:
          example_setting: "Example value"
```

Reply:
Ah, I was a bit too excited for this I guess ;) Thanks